CHANGELOG.md

```diff
@@ -5,9 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [6.2.3] - 2020-12-22
 ### Fixed
-- The Conjur CLI now raises a proper error when trying to rotate a non-existing user's API key
-  ([cyberark/conjur#979](https://github.com/cyberark/conjur/issues/979))
+- The Conjur CLI now raises a proper error when trying to rotate a non-existing
+  user's API key.
+  [cyberark/conjur#979](https://github.com/cyberark/conjur/issues/979)
```

Comment:
All other changes haven't got a new line for the issue... do we care for that?

Reply:
I'm not sure about the history of this change log file, but across our suite this is common.